playbooks/utils/common_only.yml

```diff
@@ -6,6 +6,9 @@
   # hosts: staging:qa:production
   remote_user: pulsys
   become: true
+  vars_files:
+    - ../group_vars/nginxplus/main.yml
```

Comment:
I believe this would mean that the app-protect log rotation directive would be put on all machines this playbook runs against.

Reply:
Per discussion, the above comment is true. However, we are never going to merge this change. It is only being used as a stopgap while we wait for our new nginxplus license.